src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
```

Comment:
GetArea is a good name for a method, for an interface I do with something like `CalculableArea`